src/controller/CHIPDeviceController.h

```diff
@@ -514,8 +514,10 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
                                        uint8_t option);
 
     /**
-     *  This function call indicates commissioning complete and sends commissioining complete
-     *  complete event to the application.
+     *  This function call causes the DeviceCommissioner to send a
+     *  CommissioningComplete command to the given node.  At least sometimes.
```

Comment:
"At least sometimes" ?

Can we do better than that in documentation? I would rather delete this and the following sentence if we cannot improve as this vagueness does not seem to help me when reading the comment.

Reply:
Does the below sentence clarify this?

Reply:
Will add a pointer to https://github.com/project-chip/connectedhomeip/issues/8010 for clarification, I guess.  But really, the behavior of this function is nuts (in ways that are somewhat independent of #8010) and there is no sane way to document it in its current state.  :(

Reply:
Fixed the comment to say:
```
     *  This function call causes the DeviceCommissioner to send a
     *  CommissioningComplete command to the given node.  At least when
     *  mIsIPRendezvous is false, which seems to be an incredibly broken
     *  workaround for
     *  <https://github.com/project-chip/connectedhomeip/issues/8010>.  Chances
     *  are, this function and its callsites should just be removed when that
     *  issue is fixed.
```